### PR TITLE
feat: ru locale

### PR DESCRIPTION
**Pull request by alexhooketh**
# Updating the Clave Locales

["And remember,..."](https://www.youtube.com/watch?v=OuTFxHM07oQ)

### Checklist

-   [X] I am adding a new locale
-   [ ] I am updating an existing locale
-   [ ] I have read the [contribution guidelines](../CONTRIBUTING.md) and followed the rules accordingly

### Locale details

-   **Locale name:** `ru.ts`

### CODE REVIEW

File: src/ru.ts [hunk 1]
```diff
@@ -0,0 +1,2508 @@
...
+ */
+
+export const ru = {
+    locale: "ru",
+    general: {
+        identifiers: {
+            self: "Я",
+        },
+    },
+    error_codes: {
```
Comment (asgarovf) [inline src/ru.ts:14]: These errors are also displayed on UI at some points. Could you please add those too?

File: src/ru.ts [hunk 2]
```diff
@@ -0,0 +1,2508 @@
...
+            soon: "(Скоро)",
+            portfolio: "Портфель активов",
+            onramp: {
+                title: "Банковский перевод",
+                detail: "Детали",
+                estimated_time: "Примерное время",
+                details: {
+                    accepting: "Перевод на сумму {{amount}} {{symbol}} в процессе.",
+                    purchasing: "USDC готовы к отправке.",
+                    sending: "Производится перевод USDC.",
```
Comment (asgarovf) [inline src/ru.ts:2222]: This is not important for Russian users but the onramped amount comes as USDC.e :(

```suggestion
                    accepting: "Перевод на сумму {{amount}} {{symbol}} в процессе.",
                    purchasing: "USDC.e готовы к отправке.",
                    sending: "Производится перевод USDC.e.",
```

File: src/ru.ts [hunk 3]
```diff
@@ -0,0 +1,2508 @@
...
+            home: "Домой",
+        },
+        sign_bottom_sheet: {
+            title: "Запрос на подпись",
+            description: "Подписывая это сообщение, вы разрешаете восстановление этого аккаунта как его доверенное лицо. Этот аккаунт можно будет восстановить через 48 часов после вашего подтверждения. После этого, вы сможете завершить процесс восстановления по той же ссылке.",
+            release: "Разрешено",
+            sign: "Подписать",
+        },
+        success: {
+            user_title: "Начат процесс восстановления ",
```
Comment (asgarovf) [inline src/ru.ts]:
```suggestion
            user_title: "Начат процесс восстановления",
```

File: src/ru.ts [hunk 4]
```diff
@@ -0,0 +1,2508 @@
...
+            someone_title: "Восстановление уже идёт для ",
```
Comment (asgarovf) [inline src/ru.ts]:
```suggestion
            someone_title: "Восстановление уже идёт для",
```

**Review by asgarovf — APPROVED** [review]
Looks good to me overall. Let's test on live!

File: src/ru.ts (hunk 1 above)
Comment (alexhooketh) [reply src/ru.ts:14]: on it

File: src/ru.ts (hunk 2 above)
Comment (alexhooketh) [reply src/ru.ts:2222]: This is not just "not important". This is very confusing. Especially in the areas of extensive Tron usage (i.e. USDT), such as post-soviet, where people don't even know USDC, let aside USDT.

image

image

Comment (alexhooketh) [reply src/ru.ts:2222]: FYI the only reason the second options are not at 0% is because I explained the difference when just onboarded them

**Comment by alexhooketh** [comment]
btw I migrated the old ru locale from json to ts so it could have dead fields that you might want to sanitize (remove the fields in ru.ts that are not present in en.ts)

**Pull request merged**